test/stops/repo_test.exs

```diff
@@ -1,8 +1,6 @@
 defmodule Stops.RepoTest do
-  @moduledoc """
-  TODO: Mock API data in this whole file
-  """
```

Comment:
This is my favorite.

Reply:
rofl it's a crowd pleaser!

<img width="436" alt="image" src="https://github.com/mbta/dotcom/assets/2136286/99bacb80-6c38-4181-91c7-cc0252596b98">